paraatm/tests/test_para_atm.py

```diff
@@ -106,6 +112,20 @@ def test_vcas(self):
         sim = VCAS(cfg)
         track = sim()['trajectory']
         self.assertEqual(len(track), 1000)
+    def test_aviationr(self):
+        cur_dir = os.path.dirname(os.path.abspath(__file__))
+        data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+
+        cfg = {'fp_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_GateToGate.trx',  # flight plan file
+               'mfl_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_mfl.trx',  # mfl file
+               'data_file': data_dir + 'aviationR/data/',
+               'model_file': data_dir + 'aviationR/model/',
+               'sim_time': 8000}  # total simulation time
+
+        # call
+        sim = AviationRisk(cfg)
+        track = sim()['trajectory']  # call simulation function using NatsSimulationWrapper
+        self.assertEqual(len(track), 8000)
```

Comment:
For me, this fails because `len(track)` is 267.  I believe that this is because you have set the simulation time step to 30, not 1.  It is good practice to run these tests yourself (`python -m unittest`) to verify that they work prior to submitting the changes.